operators/src/processing/expression/mod.rs

```diff
@@ -265,6 +265,17 @@ impl RasterOperator for Expression {
         let time = time_interval_extent(in_descriptors.iter().map(|d| d.time));
         let bbox = partitions_extent(in_descriptors.iter().map(|d| d.bbox));
 
+        let resolution = in_descriptors
+            .iter()
+            .map(|d| d.resolution)
+            .reduce(|a, b| match (a, b) {
+                (Some(a), Some(b)) => {
+                    Some(SpatialResolution::new_unchecked(a.x.min(b.x), a.y.min(b.y)))
```

Comment:
That is actually strange, but I have no better solution.
You would not be able to query without having some interpolation… I don't know…

Reply:
What do you mean?

The idea is taking the finest resolution of all inputs as the finest resolution of the output. This is the only thing that makes sense to me. What does it have to do with interpolation, though?

Reply:
cf. chat, we can leave it for now